src/runtime/components/forms/SelectMenu.vue

```diff
@@ -551,6 +551,10 @@ export default defineComponent({
     })
 
     function onUpdate(value: any) {
+      if (toRaw(props.modelValue) === value) {
```

Comment:
nice, I wonder would this also be a benefit to InputMenu component as well?

Reply:
Thank you for the feedback, @ineshbose! That’s a great point. I’ll review the InputMenu component to see if these changes could enhance its functionality.